### PR TITLE
Enable level0 and hypershift hosted cluster in Logging WebUI Test

### PR DESCRIPTION
**Pull request by anpingli**
By default run all Logging UI cases 
Use console-test-frontend-hypershift.sh when cluster is hypershift hosted Cluster
Run level0 cases when E2E_RUN_TAGS ~ @level0

### CODE REVIEW

File: ci-operator/step-registry/openshift-extended/logging-test/disruptive/openshift-extended-logging-test-disruptive-ref.metadata.json [hunk 1]
```diff
@@ -2,18 +2,15 @@
...
 			"QiaolingTang",
-			"kabirbhartiRH",
-			"gkarager"
+			"kabirbhartiRH"
 		]
 	}
-}
\ No newline at end of file
+}
+
```
Comment (QiaolingTang) [inline ci-operator/step-registry/openshift-extended/logging-test/disruptive/openshift-extended-logging-test-disruptive-ref.metadata.json:16]: Please don't add new line at the end of `*.metadata.json`

Comment (anpingli) [reply ci-operator/step-registry/openshift-extended/logging-test/disruptive/openshift-extended-logging-test-disruptive-ref.metadata.json]: Fixed.
```
1) check if there is newline at the end. cat -e <file>
2) remove the newline at the end of the file  truncate -s -1 file
```

**Review by IshwarKanse — APPROVED** [review]

File: ci-operator/step-registry/openshift-extended/logging-test/web-tests/openshift-extended-logging-test-web-tests-commands.sh [hunk 2]
```diff
@@ -17,8 +17,26 @@ fi
...
+    run_shell="console-test-managed-service.sh"
+fi
+## determine if it is hypershift guest cluster or not
+if ! (oc get node --kubeconfig=${KUBECONFIG} | grep master) ; then
+    run_shell="console-test-frontend-hypershift.sh"
+fi
+
+if [[ $E2E_RUN_TAGS =~ @level0 ]]; then
+    echo "only run level0 scenarios"
+    ./${run_shell} ./tests/logging/--tags @level0 || true
```
Comment (QiaolingTang) [inline ci-operator/step-registry/openshift-extended/logging-test/web-tests/openshift-extended-logging-test-web-tests-commands.sh:35]:
```suggestion
    ./${run_shell} ./tests/logging/ --tags @level0 || true
```

Comment (anpingli) [reply ci-operator/step-registry/openshift-extended/logging-test/web-tests/openshift-extended-logging-test-web-tests-commands.sh]: Fixed

**Pull request merged**